src/js/helpers/timeRangeHelper.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * timeRangeHelper.js
+ * Created by Lizzie Salita 3/8/18
+ */
+
+import moment from 'moment';
+
+export const convertDatesToRange = (startDate, endDate) => {
+    if ((startDate && endDate) && (moment.isMoment(startDate) && moment.isMoment(endDate))) {
+        const duration = moment.duration(endDate.diff(startDate));
+        const years = duration.years();
+        const months = duration.months();
+        let yearString = '';
+        let monthString = '';
+
+        if (!isNaN(years)) {
+            if (months > 0) {
+                if (months === 1) {
+                    monthString = `${months} month`;
+                }
+                else {
+                    monthString = `${months} months`;
+                }
+            }
+
+            if (years > 0) {
+                if (years === 1) {
+                    yearString = `${years} year`;
+                }
+                else {
+                    yearString = `${years} years`;
+                }
+            }
+
+            if (monthString && yearString) {
+                return `(${yearString}, ${monthString})`;
+            }
+
+            return `(${monthString}${yearString})`;
+        }
+        return '';
```

Comment:
This doesn't seem to catch durations of less than 1 month. Currently durations of less than 1 month return empty parens `()`.

Reply:
Look at `#/award/612`